egs/librispeech/ASR/pruned_transducer_stateless2/scaling.py

```diff
@@ -377,6 +378,155 @@ def forward(self, input: Tensor) -> Tensor:
         return self._conv_forward(input, self.get_weight())
 
 
+class ScaledLSTM(nn.LSTM):
+    # See docs for ScaledLinear.
+    # This class implements LSTM with scaling mechanism, using `torch._VF.lstm`
+    # Please refer to https://github.com/pytorch/pytorch/blob/master/torch/nn/modules/rnn.py
+    def __init__(
+        self,
+        *args,
+        initial_scale: float = 1.0,
+        initial_speed: float = 1.0,
+        **kwargs
+    ):
+        # Hardcode bidirectional=False
```

Comment:
I suggest using
```python3
if 'bidirectional' in kwargs:
  assert kwargs['bidirectional'] is False
```
`bidirectional` is `False` by default, so you don't need to pass it below.